internal/utils.go

```diff
@@ -598,5 +598,10 @@ func getRootRemediationAction(policyTemplates []map[string]interface{}) string {
 		}
 	}
 
+	// "InformOnly" should only apply to ConfigurationPolicies
+	if action == "InformOnly" {
```

Comment:
It seems we accept any casing so this should also accept that too:
https://github.com/open-cluster-management-io/governance-policy-framework-addon/blob/6a8baf33b678533fb6ebe268edd47a421cc81143/controllers/templatesync/template_sync.go#L1345